art/params.py

```diff
@@ -15,8 +16,23 @@
 TEXT_LARGE_THRESHOLD = 7  # pragma: no cover
 TEXT_MEDIUM_THRESHOLD = 10  # pragma: no cover
 
+DEFAULT_DELAY = 0.5  # pragma: no cover
+
+
+class PrintingMode(Enum):
```

Comment:
I recommend eliminating this `Enum` and using `str` for better consistency. We will introduce `Enum` support in a future release.